### PR TITLE
Docs tweaks

### PR DESCRIPTION
**Pull request by MartijnCuppens**
- Simplify display headings example markup
- Simplify footer styles
- Remove z-indices hacks in the docs
- Remove redundant modal styling in docs
- Use `escape-svg()` to escape docs sidebar chevron
- Simplify sidebar chevron code
- Cleanup masthead css & fix column width between `md` & `lg`
- Easier to download logo's with the `download` attribute.
- Changed some color codes into variables
- Cleanup brands css

TODO:
- [x] Check color contrasts
- [x] Clarify how to download the logos

https://deploy-preview-29347--twbs-bootstrap.netlify.com/

### CODE REVIEW

File: site/content/docs/4.3/content/typography.md [hunk 1]
```diff
@@ -105,22 +105,10 @@ Use the included utility classes to recreate the small secondary heading text fr
 Traditional heading elements are designed to work best in the meat of your page content. When you need a heading to stand out, consider using a **display heading**—a larger, slightly more opinionated heading style. Keep in mind these headings are not responsive by default, but it's possible to enable [responsive font sizes](#responsive-font-sizes).
 
 <div class="bd-example bd-example-type">
```
Comment (XhmikosR) [inline site/content/docs/4.3/content/typography.md]: is `bd-example-type` still needed here?

**Comment by XhmikosR** [comment]
The only concern I have is the z-index changes. The rest mostly LGTM.

File: site/content/docs/4.3/content/typography.md (hunk 1 above)
Comment (MartijnCuppens) [reply site/content/docs/4.3/content/typography.md]: Nope, removed it

**Comment by MartijnCuppens** [comment]
Pushed some more changes for the masthead. The `z-index` changes should be fine. @mdo, could you confirm this?

**Comment by XhmikosR** [comment]
The clipboard hiding in the homepage was intentional. We had issues with
the button overlapping because these snippets are one line only

**Comment by XhmikosR** [comment]
Can you add the preview URL in the PR?

**Comment by MartijnCuppens** [comment]
> The clipboard hiding in the homepage was intentional.

There's no `.masthead-followup` class, the buttons are currently also shown on the homepage.

**Comment by XhmikosR** [comment]
Hmm then in that case we need to confirm if there's an issue with the copy
button on IE, Safari etc. This sneaked in that forms PR I believe

**Comment by XhmikosR** [comment]
It seems that IE11 on hover shows the horizontal bar thus overlaps with content. I remember we had an issue about this, not sure what happened with that.

ie11

Not sure how to fix that, but it's not a regression because it's present on <https://getbootstrap.com/> too.

So, I'd say the copy buttons are fine I guess.

**Comment by XhmikosR** [comment]
I pushed a couple of patches too. We might be able to simplify _brand.scss too, but I'm not sure what's the line in using utils.

**Comment by MartijnCuppens** [comment]
Simplified `_brand.scss` a bit. Not sure what to do with that scrollbar issue, I noticed the copy button is still clickable, so the current situation is acceptable for me.

**Comment by XhmikosR** [comment]
@MartijnCuppens: Did you check if the color changes are WCAG2AA valid? You can use https://contrast-ratio.com/

File: site/content/docs/4.3/about/brand.md [hunk 2]
```diff
@@ -31,25 +31,25 @@ Use either the Bootstrap mark (a capital **B**) or the standard logo (just **Boo
...
-    <img class="svg" src="/docs/{{< param docs_version >}}/assets/brand/bootstrap-solid.svg" alt="Bootstrap" width="144" height="144">
-  </div>
-  <div class="bd-brand-item inverse">
-    <img class="svg" src="/docs/{{< param docs_version >}}/assets/brand/bootstrap-outline.svg" alt="Bootstrap" width="144" height="144">
-  </div>
-  <div class="bd-brand-item inverse">
-    <img class="svg" src="/docs/{{< param docs_version >}}/assets/brand/bootstrap-punchout.svg" alt="Bootstrap" width="144" height="144">
-  </div>
+<div class="bd-brand-logos d-sm-flex text-center bg-light rounded overflow-hidden w-100 mb-3">
+  <a href="/docs/{{< param docs_version >}}/assets/brand/bootstrap-solid.svg" download class="bd-brand-item">
```
Comment (XhmikosR) [inline site/content/docs/4.3/about/brand.md]: Maybe we should add `title`s with "Click to download logo"?

File: site/content/docs/4.3/about/brand.md [hunk 3]
```diff
@@ -31,25 +31,25 @@ Use either the Bootstrap mark (a capital **B**) or the standard logo (just **Boo
 
 ## Download mark
 
-Download the Bootstrap mark in one of three styles, each available as an SVG file. Right click, Save as.
+Download the Bootstrap mark in one of three styles, each available as an SVG file. Click to download the logos.
```
Comment (XhmikosR) [inline site/content/docs/4.3/about/brand.md]: Or make the new phrase bold maybe. Not sure it's good, BTW. Maybe it should say "Click on each logo to download it." or something. Let's wait for someone else's feedback.

**Comment by MartijnCuppens** [comment]
> @MartijnCuppens: Did you check if the color changes are WCAG2AA valid? You can use contrast-ratio.com

Yup, had to change 2 colors (clipboard & carbonads)

**Comment by XhmikosR** [comment]
LGTM. I'd like one approval from @ysds about the modal and z-index changes.

BTW there's a [new issue](https://github.com/twbs/bootstrap/pull/29368#issuecomment-528725686), maybe you could sort it out here too?

**Comment by MartijnCuppens** [comment]
The new issue is fixed. I've also removed the support query, since in this case it is redundant.

**Review by XhmikosR — APPROVED** [review]

**Comment by ysds** [comment]
> LGTM. I'd like one approval from @ysds about the modal and z-index changes.

i’m gonna see later :)

**Review by ysds — APPROVED** [review]

**Comment by XhmikosR** [comment]
Love this cleanup ❤️ Let's confirm we didn't break anything :P

**Pull request merged**